### PR TITLE
[IMP] *: remove makeFakeUIService

### PR DESCRIPTION
**Pull request by Polymorphe57**

### CODE REVIEW

**Comment by ged-odoo** [comment]
robodoo r+

**Comment by aab-odoo** [comment]
robodoo r-

File: addons/web/static/tests/webclient/actions/legacy_tests.js [hunk 1]
```diff
@@ -141,6 +141,9 @@ QUnit.module("ActionManager", (hooks) => {
         assert.containsOnce(webClient, ".custom-action");
         assert.verifySteps([]);
 
+        // close debug menu
+        await click(webClient.el, ".o_debug_manager button");
+        // open debug menu
```
Comment (aab-odoo) [inline addons/web/static/tests/webclient/actions/legacy_tests.js:146]: is it related?

Comment (Polymorphe57) [reply addons/web/static/tests/webclient/actions/legacy_tests.js:146]: yes, with the "true" ui service, the debug manager stays open. So we have to close it before reopening it

Comment (Polymorphe57) [reply addons/web/static/tests/webclient/actions/legacy_tests.js:146]: this is due to the fact that the computation of activeElement was not done correctly in the "fake" service (activateElement, ... not correct)

**Comment by aab-odoo** [comment]
robodoo r+

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request merged**